### DISCUSSION TITLE
RNP for Chapelo Airport in Argentine (SAZY)

### DISCUSSION THREAD

**Discussion by marloscaida** [albar965/littlenavmap#895] [General]
Greetings to all participants.
Is there any chance to know the decoding of the comma separated fields of the files located in the x-plane C:\X-Plane 11\Resources\default data\CIFP folder?

They are text files with a .DAT extension and contain the SID/STAR/APP procedures for each airport.

I suppose that in order to build this wonderful software, Little NavMap, you had to be able to access this knowledge.

I know that these files are created and generated by Jeppsen, but I cannot access this information.

My humble project is to add two RNP procedures for the Chapelco Airport (SAZY) in Argentina, based on the private letters of Aerolineas Argentinas, the only one approved for the use of these procedures.

I'm doing it with a text editor, using reverse engineering but without much success.

Thanks in advance to whoever can provide me with this information.

**Comment by albar965** [comment]
[X-Plane CIFP documentation](https://developer.x-plane.com/wp-content/uploads/2019/01/XP-CIFP1101-Spec.pdf) from [here](https://developer.x-plane.com/article/navdata-in-x-plane-11/).

They are referring to the official ARINC 424 documentation chapters from this document which costs ridiculous money but there is an older ARINC 424-17 PDF around in the webs.

Alex

**Reply by marloscaida** [reply]
Alex, Thank you very much for your prompt response. I already managed to get the document..
Regards 
carlos

**Comment by albar965** [comment]
You're welcome. :slightly_smiling_face:
